auraescript/macros/src/ops.rs

```diff
@@ -166,7 +166,7 @@ fn typescript_generator(input: &OpsGeneratorInput) {
 
     let ts_path = {
         let mut out_dir = gen_dir.clone();
-        out_dir.push(format!("v0/{module}.ts"));
+        out_dir.push(format!("runtime/{module}/v0/{module}.ts"));
```

Comment:
This isn't the correct way to fix the macro as we have more modules than `runtime`, but if it works in this PR and you are not comfortable making the change, I can fix it after if we merge this as is.